samples/tfm_integration/tfm_psa_arch_tests/sample.yaml

```diff
@@ -5,4 +5,5 @@ sample:
 tests:
   sample.tfm_ipc:
     tags: introduction
-    platform_whitelist: mps2_an521_nonsecure v2m_musca_b1_nonsecure
```

Comment:
I am not sure if the CI care about this build time strictly? I think this platform is good to build while CI check since this sample should support on Musca. Besides, the Zephyr PSA L1 was certified on Musca B1.

Reply:
Is there a specific benefit to running these long tests (750MB download for the TF-M repo and dependencies) twice? It seems like everything is tested already on the MPS2+ which is the defacto M33 qemu platform, and running musca as well would just add overhead without any clear benefit since the two devices and sample configs are functionally equivalent (the Musca B1 is just a silicon implementation of the code running on the FPGA on the MPS2+ w/AN521?